src/main/java/io/confluent/kafkarest/KafkaRestConfig.java

```diff
@@ -129,6 +140,12 @@
           + " Use 0 for no timeout";
   public static final String SIMPLE_CONSUMER_POOL_TIMEOUT_MS_DEFAULT = "1000";
 
+  public static final String SIMPLE_CONSUMER_MAX_POLL_TIME_CONFIG = "simpleconsumer.max.poll.time";
```

Comment:
@cpuchakayaka Hmm, that actually wasn't what I was suggesting. The producer and consumer APIs for the Java clients aren't intended to be RESTful and I don't think the mapping has to be direct. I actually think having the simple consumer style reads on the `TopicsResource` or `PartitionsResource` is quite intuitive for users and probably the right thing to do, whereas consumer groups really are a special type of resource that operate differently. I was just suggesting that user-facing settings that include the "simple" name will eventually be confusing as the old consumer APIs will eventually be deprecated and removed and newer users won't understand what "simple" means in this context.